diff --git a/Makefile b/Makefile
index 4610992..a07e5dc 100644
--- a/Makefile
+++ b/Makefile
@@ -62,7 +62,8 @@ install-bin: snownews
 	$(INSTALL) -s snownews $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) opml2snow $(DESTDIR)$(PREFIX)/bin
 	if [ ! -f "$(DESTDIR)$(PREFIX)/bin/snow2opml" ]; then \
-		ln -sf $(DESTDIR)$(PREFIX)/bin/opml2snow $(DESTDIR)$(PREFIX)/bin/snow2opml; \
+		(cd $(DESTDIR)$(PREFIX)/bin && \
+		 ln -sf opml2snow snow2opml ); \
 	fi;
 	$(INSTALL) snowsync $(DESTDIR)$(PREFIX)/bin
 
